cron/internal/data/projects.csv

```diff
@@ -838358,11 +838358,12 @@ github.com/metal/metal.js,num_dependents_deps.dev:778
 github.com/metal/soyparser,
 github.com/metal3-io/baremetal-operator,num_dependents_deps.dev:4
 github.com/metal3-io/cluster-api-provider-metal3,num_dependents_deps.dev:0
+github.com/metal3-io/community,num_dependents_deps.dev:0
 github.com/metal3-io/hardware-classification-controller,num_dependents_deps.dev:0
 github.com/metal3-io/ip-address-manager,num_dependents_deps.dev:0
-github.com/metal3-io/ironic-client,num_dependents_deps.dev:0
```

Comment:
i'm going to wait to merge this until the archive status is picked up by the cron